endpoints.json

```diff
@@ -86,5 +86,35 @@
         }
       ]
     }
+  },
+  "POST /api/articles/:article_id/comments": {
+    "description": "serves the new comment for a specific article based on requested article_id",
+    "queries": [],
+    "exampleResponse": {
+      "comment": {
+        "comment_id": 19,
+        "body": "Don't you just LOVE commenting?",
+        "article_id": 3,
+        "author": "butter_bridge",
+        "votes": 0,
+        "created_at": "2024-11-26T12:33:05.964Z"
+      }
+    }
+  },
+  "PATCH /api/articles/:article_id": {
+    "description": "serves an updated article based on requested article_id and inc_votes",
+    "queries": [],
+    "exampleResponse": {
+      "article": {
+        "article_id": 1,
+        "title": "Living in the shadow of a great man",
+        "topic": "mitch",
+        "author": "butter_bridge",
+        "body": "I find this existence challenging",
+        "created_at": "2020-07-09T20:11:00.000Z",
+        "votes": 105,
+        "article_img_url": "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700"
+      }
```

Comment:
Nice, but dont forget to include the `"expectedBody" : { inc_votes: 1 }`, so we know what this patch requests should be expected to send. (Same for the above post)